src/encoding/address.js

```diff
@@ -41,7 +41,12 @@ function isValidAddress(address) {
     return utils.arrayEqual(checksum, decoded.checksum);
 }
 
-function decode(address) {
```

Comment:
These specific functions were used internally as `address.decode/encode` but exported by `main.js` in wrapper functions `decodeAddress/encodeAddress`. As part of the refactor I did, I thought it would be simpler to give them their public names from the beginning to avoid renaming later. So the end user shouldn't notice a difference here, unless they specifically import the internal file `algosdk/src/encoding/address`.

In JS it's best practice for modules to export all public functions & variables from their main entrypoint (`main.js` in our case), so I don't think we should support users importing our internal files. These internal files aren't even accessible in the browser from the bundle we build, so it would be unfair to expect users to have to do that anyway to access parts of the SDK.